rfc/0009-policy-report.md

```diff
@@ -0,0 +1,323 @@
+
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Policy Report                           |
```

Comment:
```suggestion
| Feature Name | Policy Report  for Audit background checks                         |
```
Or similar. I just want to make clear that it is about the background checks that simulate requests instead of logging real requests.

Reply:
Done